test/unit_test/utils.py

```diff
@@ -162,23 +144,31 @@ def get_mock_openvino_model():
     return olive_model
 
 
-def create_dataloader(data_dir, batch_size, *args, **kwargs):
-    return DataLoader(DummyDataset(1))
-
-
-def create_fixed_dataloader(data_dir, batch_size, *args, **kwargs):
-    return DataLoader(FixedDummyDataset(1))
+def _get_dummy_data_config(name, input_shapes, input_names=None, input_types=None, size=1, batch_size=1):
+    data_config = DataConfig(
+        name=name,
+        type="DummyDataContainer",
+        load_dataset_config=DataComponentConfig(
+            params={
+                "input_shapes": input_shapes,
+                "input_names": input_names,
+                "input_types": input_types,
+                "size": size,
+            }
+        ),
+        dataloader_config=DataComponentConfig(params={"batch_size": batch_size}),
```

Comment:
batch size is ignored here too. will probably have to update the callers to provide the correct shape?

Reply:
Fixed